### DISCUSSION TITLE
Fleet policy and output for integrations

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#11035] [2.4] [answered]
Hi!

SO 2.4.5, distributed.
I'd like to change output for both default fleet policy aka "endpoints-initial" and custom policies to heavy node's logstash. So I added new output in fleet settings, but changing integration output for both kinds of policies is blocked, only available option is  default grid-logstash.
What am I doing wrong?

**Comment by senatesan** [comment, marked as answer]
I have noticed that this is a limitation of the Basic Elastic Subscription included with SO. Elastic calls this feature Per Policy Output Assignment. 

https://www.elastic.co/guide/en/fleet/current/agent-policy.html#change-policy-output
https://www.elastic.co/subscriptions

**Reply by Sahale** [reply]
You are totally right.
A little bit unclear, how heavy nodes are going to work now.

**Comment by TOoSmOotH** [comment]
You are not doing anything wrong. Elastic has locked multiple endpoint policies behind their premium paid licensing. We are in the process of updating the documentation but one way to force certain elastic agent to specific receiver nodes is via firewall rules. The agent will get a list of all receiver nodes and then start down the list. The first one it connects to it uses. So in theory you could block all the ones that you don't want the agents to connect to.

**Reply by Sahale** [reply]
Ok, I see.
Main problem is that now with SO 2.3 PC's and servers in different offices don't have direct connection with Manager node, therefore with 2.4 agents won't be able to connect to Fleet and get actual policy and settings.
I am trying to find out, if we can bypass it with Fleet nodes in every office. But it's question from [another discussion](https://github.com/Security-Onion-Solutions/securityonion/discussions/11010)
Thanks!
